tests/Panel/PanelTest.php

```diff
@@ -635,6 +687,52 @@ public function testGlobalsWithUser(): void
         $this->assertSame('de', $globals['$translation']['code']);
     }
 
+    /**
+     * @covers ::go
+     */
+    public function testGo()
+    {
+        try {
+            Panel::go('test');
+        } catch (Redirect $r) {
+            $this->assertSame('/panel/test', $r->getMessage());
+            $this->assertSame(302, $r->getCode());
+        }
```

Comment:
To ensure that the exception is thrown at all:

```suggestion
        $thrown = false;

        try {
            Panel::go('test');
        } catch (Redirect $r) {
            $thrown = true;
            $this->assertSame('/panel/test', $r->getMessage());
            $this->assertSame(302, $r->getCode());
        }

        $this->assertTrue($thrown);
```